src/components/pages/Book.js

```diff
@@ -4,7 +4,22 @@ import '../styles/Book.css';
 
 const Book = () => (
   <div className="book-div">
-    <h1>Here goes the book</h1>
+    <li className="book">
+      <span className="title">The Lord of The Rings (JRR Tolkien)</span>
+      <button type="button">Delete</button>
+    </li>
+    <li className="book">
+      <span className="title">O Auto da Compadecida (Ariano Suassuna)</span>
+      <button type="button">Delete</button>
+    </li>
+    <li className="book">
+      <span className="title">Mininos Sabidos (J. Teixeira)</span>
+      <button type="button">Delete</button>
+    </li>
+    <li className="book">
+      <span className="title">Crime and Punishment (Fiodor Dostoyevsky)</span>
+      <button type="button">Delete</button>
```

Comment:
- [ ] the `Book` component should Take the book data as a prop and renders the title and the author, this means it should be passed down from the parent component